server/game/cards/02_SHD/units/OutlawCorona.ts

```diff
@@ -0,0 +1,22 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class OutlawCorona extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '3503780024',
+            internalName: 'outlaw-corona'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addBountyAbility({
+            title: 'Put the top card of your deck into play as a resource.',
+            targetResolver: {
```

Comment:
I think you can omit targetResolver and write only immediateEffect

Reply:
Yeah, great catch! Fixed it.